pkg/apis/migration/v1alpha1/labels.go

```diff
@@ -1,15 +1,17 @@
 package v1alpha1
 
 import (
+	"strings"
+
 	migref "github.com/konveyor/mig-controller/pkg/reference"
 	"k8s.io/apimachinery/pkg/types"
-	"strings"
 )
 
 // Labels
 const (
-	PartOfLabel = "app.kubernetes.io/part-of" // = Application
-	Application = "openshift-migration"
+	PartOfLabel     = "app.kubernetes.io/part-of" // = Application
+	Application     = "openshift-migration"
+	MigratedByLabel = "migration.openshift.io/migrated-by" // (migmigration UID)
```

Comment:
A unique label is set to select only pieces of migrated application by label selector.